source/components/GlobalNavigation/hooks/useLazyLoad.js

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useRef } from 'react';
```

Comment:
Seems universal enough to live in `/hooks/` - please move it there and add it to the styleguide with proper instructions.

Reply:
https://github.com/Wikia/react-common/pull/22/commits/8b3fabf022e2ebf2d6b673f37facc4ec760992d2